### DISCUSSION TITLE
Literature Survey

### DISCUSSION THREAD

**Discussion by Pranav400-source** [Tiham2001/tarp#4] [General]
# **Add few literature reviews**
- Please try to find some past works regarding our project.
- Remember the recent works should be related machine learning and career path prediction.

## Things to keep in mind
- The literature survey must contain papers not older than 3 years.
- Information elicitation on - 
  - Name of the paper
  - Date of publication of the paper
  - Work done in the paper
  - Any gaps or limitations in the existing strategies adopted by the paper
### **_Please discuss any doubts regarding the addition and validation of the paper in this._**

**Comment by Pranav400-source** [comment]
I got a paper regarding out topic.
Please validate it.
Paper name - A Comparative Study of Machine Learning Algorithms for Career Prediction

**Reply by Tiham2001** [reply]
Okay it is perfect!!👍

**Reply by annanya1803** [reply]
Okay it works! I will also try to find for more papers

**Comment by Tiham2001** [comment]
Added three research papers. Please do verify.
Thank You..

**Reply by Pranav400-source** [reply]
Papers seem okay, try writing a descriptive work done for the first paper.
Rest is fine.

**Reply by Tiham2001** [reply]
Yes I will do that definitely

**Comment by Pranav400-source** [comment]
Please check my recent commit, I have added three more papers. Let me know of any modifications in them.

**Reply by annanya1803** [reply]
The papers look fine. Good work

**Comment by annanya1803** [comment]
I have added 2 new research papers. Please check it and let me know

**Reply by Pranav400-source** [reply]
Great!

**Reply by Tiham2001** [reply]
Looks good

**Comment by shalinimishra31** [comment]
I have added two more paper. Have a look at them once.

**Reply by Tiham2001** [reply]
Yes they are absolutely perfect 🤩

**Reply by annanya1803** [reply]
Good job!
